open-models/serving/cloud_run_ollama_gemma2_rag_qa.ipynb

```diff
@@ -0,0 +1,1007 @@
+{
```

Comment:
<p>Perhaps add something afetr the "many ways to deploy" sentence, like:</p><p>"You can start with the recommended settings below as a starting point."</p><p> </p><p>Or perhaps something along the lines of "The settings below show the minimum ____ needed to run Gemma 2, but you can increase the X, Y, Z if you want more performance (latency, throughput)."</p><p> </p><p>Rationale: just looking to help minimize user confusion with too many options, and give best practices where we can.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/968/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
good point, I clarified this part adding what you suggested.